src/main/java/com/avengers/gamera/service/ArticleService.java

```diff
@@ -0,0 +1,30 @@
+package com.avengers.gamera.service;
+
+import com.avengers.gamera.dto.article.ArticleGetDto;
+import com.avengers.gamera.dto.article.ArticlePostDto;
+import com.avengers.gamera.entity.Article;
+import com.avengers.gamera.mapper.ArticleMapper;
+import com.avengers.gamera.repository.ArticleRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class ArticleService {
+    private final ArticleRepository articleRepository;
+    private final ArticleMapper articleMapper;
+
+    public ArticleGetDto createArticle(ArticlePostDto articlePostDto) {
+
+        Article article = articleMapper.articlePostDtoToArticle(articlePostDto);
+        String img = article.getCoverImgUrl();
+
+        //TODO after image upload function implemented, this url should from user
+        if (img == null || img.isEmpty()) {
+            article.setCoverImgUrl("https://spicsum.photos/800/400");
```

Comment:
I can be set as default value, this step is unnecessary.

Reply:
In which layer to set the default value?  
Btw this is a temporary value, which will be removed later.